src/riscv_machine.cpp

```diff
@@ -681,7 +681,7 @@ void fdt_end(FDTState *s)
 
 static int riscv_build_fdt(RISCVMachine *m, uint8_t *dst, const char *dtb_name, const char *cmd_line)
 {
-    FDTState *s;
+    FDTState *s = 0;
```

Comment:
It is fine, but we may want to start to converge to use nullptr for a the null/0 pointers.

Reply:
Noted.  I'll fix eventually.